### PR TITLE
[BPK-1974] Switch component

### PR DESCRIPTION
**Pull request by tiagoengel**

### CODE REVIEW

File: Backpack/src/main/java/net/skyscanner/backpack/toogle/BpkSwitch.kt [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package net.skyscanner.backpack.toogle
```
Comment (shaundon) [inline Backpack/src/main/java/net/skyscanner/backpack/toogle/BpkSwitch.kt]:
```suggestion
package net.skyscanner.backpack.toggle
```

Will need to be changed across the package

Comment (tiagoengel) [reply Backpack/src/main/java/net/skyscanner/backpack/toogle/BpkSwitch.kt]: 🤦‍♂️

Comment (tiagoengel) [reply Backpack/src/main/java/net/skyscanner/backpack/toogle/BpkSwitch.kt]: Done

**Comment by shaundon** [comment]
Can't get it running locally, not sure what this means. I can look into it further once I'm back.

screen shot 2018-10-26 at 14 51 43

**Comment by tiagoengel** [comment]
Just checked and it works fine here. I can look into it with you

**Pull request merged**